tests/test_integration_service.py

```diff
@@ -332,15 +332,15 @@ def test_orgunit(self):
         self.load_sample_structures(minimal=True)
 
         with self.subTest('invalid'):
-            self.assertRequestFails(
+            self.assertRequestResponse(
                 '/service/ou/00000000-0000-0000-0000-000000000000/',
-                404,
+                [],
```

Comment:
If we return an empty list on an invalid UUID, the docstring for the endpoint should be updated accordingly.